tests/integration/test_fim/test_files/test_file_limit/data/wazuh_conf_delete_full.yaml

```diff
@@ -13,13 +13,11 @@
         value: TEST_DIRECTORIES
         attributes:
         - FIM_MODE
-    - db_entry_limit:
+    - file_limit:
         elements:
           - enabled:
               value: 'yes'
-          - files:
-              value: LIMIT
-          - registries:
+          - entries:
               value: LIMIT

```

Comment:
The entries option is added. Did not add registry_limit section since this files are not related to registry configuration, so having it here is not relevant.